src/test/java/org/openmrs/contrib/qaframework/automation/ManageUserAccountSteps.java

```diff
@@ -0,0 +1,136 @@
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.reference.page.ManageUserAccountPage;
+import org.openmrs.reference.page.SystemAdministrationPage;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class ManageUserAccountSteps extends Steps {
+
+	private static final String SYSTEM_ALERT = "Atleast 8 character(s) are required";
+	private SystemAdministrationPage systemAdministrationPage;
+	private ManageUserAccountPage userAccountPage;
+
+	@Before(RunTest.HOOK.SELENIUM_USER_ACCOUNT)
+	public void visitHomePage() {
+		initiateWithLogin();
+	}
+
+	@After(RunTest.HOOK.SELENIUM_USER_ACCOUNT)
+	public void destroy() {
+		quit();
+	}
+
+	@Given("a user clicks on System Administartion app from home page")
+	public void loadSystemAdministrationPage() {
+		systemAdministrationPage = homePage.goToSystemAdministrationPage();
+	}
+
+	@Then("the system loads the system administration page")
+	public void systemloadsSystemAdministrationPage() {
+		assertTrue(textExists("System Administration"));
+	}
+
+	@When("a user clicks on Manage accounts app")
+	public void loadManageAccountsPage() {
+		userAccountPage = systemAdministrationPage.goToManageAccounts();
+	}
+
+	@Then("the system loads the manage acccount page")
+	public void systemloadsManageAccountsPage() {
+		assertTrue(textExists("Manage Accounts"));
+	}
+
+	@When("a user clicks Add New Account button")
+	public void goToAddNewAccountPage() {
+		userAccountPage.clickOnAddUser();
+	}
+
+	@Then("the system loads the add new account form")
+	public void systemLoadsAddNewAccountPage() {
+		assertTrue(textExists("Add New Account"));
+	}
+
+	@When("a user fills details of the Data clerk in the user account form")
+	public void registerDataClerkDetails() {
+		userAccountPage.enterPersonalDetails("Data", "Clerk");
+		userAccountPage.selectGender();
+		userAccountPage.clickOnAddUserAccount();
+		userAccountPage.enterUsername("dclerk");
+		userAccountPage.setUserPrivilegeLevel("High");
+		userAccountPage.setUserPassword("Clerk123", "Clerk123");
+		assertFalse(userAccountPage.containsText(SYSTEM_ALERT));
+		List<String> validationErrors = userAccountPage.getValidationErrors();
+		assertTrue(userAccountPage.isConfirmPasswordMatching(validationErrors));
+		userAccountPage.selectConfiguresForms();
+		userAccountPage.selectRecordsAllergies();
+		userAccountPage.selectRegistersPatients();
+		userAccountPage.selectUsesPatientSummary();
+		userAccountPage.addProviderDetails();
+		userAccountPage.setUserIdentifier("C010");
+		userAccountPage.setUserProviderRole("Clerk");
+	}
+
+	@When("a user fills details of the Nurse in the user account form")
+	public void registerNurseDetails() {
+		userAccountPage.enterPersonalDetails("Super", "Nurse");
```

Comment:
It would look so clean when its written as ("Super", "Nurse") .Would you take a look.

Reply:
ohh thanks , Leave one space between like ("Super", "Nurse")

Reply:
Git is funny, it shows as if there are more than one space between the text  ("Super", "Nurse") but when in the actual sense its only one space present.